libogc/console.c

```diff
@@ -311,17 +394,20 @@ void __console_init_ex(void *conbuffer,int tgt_xstart,int tgt_ystart,int tgt_str
 	con->con_stride = con_stride;
 	con->con_cols = con_xres / FONT_XSIZE;
 	con->con_rows = con_yres / FONT_YSIZE;
-	con->cursor_row = 0;
-	con->cursor_col = 0;
-	con->saved_row = 0;
-	con->saved_col = 0;
+	con->cursorY = 0;
+	con->cursorX = 0;
+	con->prevCursorX = 0;
+	con->prevCursorY = 0;
 
 	con->font = console_font_8x16;
 
-	con->foreground = COLOR_WHITE;
-	con->background = COLOR_BLACK;
+	con->fg = 7;
```

Comment:
what is 7, 0 and the flags 0?

Reply:
I can answer that- flags 0 means there are no modifications to text done besides color from table, and 0 and 7 are indices into the color table, 0 for black and 7 for white. I absolutely acknowledge that it's not the clearest system (and probably could do with a comment) since fg and bg are sometimes table color and other times a straight-up color value, but in my defense, that's just how it was before I got to the 3DS version.

Reply:
@DacoTaco I think if there are standard values for these in OS headers we should use those rather than just inventing defines

Reply:
oh ye, if the values are available in OS headers those are indeed preferred !
my problem is just about magic values hehe